sdk/objectanchors/Azure.MixedReality.ObjectAnchors.Conversion/tests/ObjectAnchorsConversionClientLiveTests.cs

```diff
@@ -0,0 +1,161 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.IO;
+using System.Numerics;
+using System.Reflection;
+using System.Threading.Tasks;
+using Azure.Core;
+using Azure.Core.TestFramework;
+using Azure.MixedReality.Authentication;
+using Azure.Storage.Blobs;
+using Azure.Storage.Blobs.Models;
+using NUnit.Framework;
+
+namespace Azure.MixedReality.ObjectAnchors.Conversion.Tests
+{
+    public class ObjectAnchorsConversionClientLiveTests : RecordedTestBase<ObjectAnchorsConversionClientTestEnvironment>
+    {
+        private const string assetsFolderName = "Assets";
+        private const string assetsFileName = "switchgear02_obj.obj";
+        private const string modelDownloadFileName = "switchgear02_obj_model.ply";
+        private const float assetGravityX = 0;
+        private const float assetGravityY = -1;
+        private const float assetGravityZ = 0;
+        private const float assetScale = 0.001f;
+
+        private static string currentWorkingDirectory => Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location);
+        private static readonly string assetLocalFilePath = Path.Combine(currentWorkingDirectory, assetsFolderName, assetsFileName);
+        public string modelDownloadLocalFilePath => Path.Combine(currentWorkingDirectory, modelDownloadFileName);
+
+        public ObjectAnchorsConversionClientLiveTests(bool isAsync)
+            : base(isAsync)
+        {
+            //TODO: https://github.com/Azure/autorest.csharp/issues/689
+            TestDiagnostics = false;
```

Comment:
The linked issue is closed. No need to disable diagnostics.

Reply:
Removed

Reply:
Removed